### DISCUSSION TITLE
Enabling multiple domains on Postal with a root domain causing issues with rDNS and HELO

### DISCUSSION THREAD

**Discussion by Praashanna** [postalserver/postal#3116] [Help with using Postal] [answered]
Can I get help with this: I was setting up a common mail server for multiple domains and ran into the issue that the helo is sending wrong domain name (not even domain name) which compares to the rDNS so, FCR DNS is not satisfied. Here's what I mean...
![Screenshot 2024-08-29 142926](https://github.com/user-attachments/assets/b40ec20d-0c25-4b7b-b432-9500cdc9bb9b)

**Comment by willpower232** [comment]
You can customise this with https://github.com/postalserver/postal/blob/da90e75036c27482699921613d838f4058a100e7/doc/config/yaml.yml#L143 (or DNS_HELO_HOSTNAME if you are using environment variables) if you need

**Comment by Praashanna** [comment]
Does changing that changes it for all the organizations (and domains)?? 

Im setting a single mail server for multiple domains using IP pools..

**Reply by willpower232** [reply]
It does not need to as the messages sent by Postal are normally tagged by the return path address so it should be fine to match on that I believe.

**Reply by Praashanna** [reply]
still couldn't get FCR DNS to work, I think it is important for a better send rate, but I cannot get it to work.

**Comment by Praashanna** [comment, marked as answer]
I got it working—I forgot to change the hostname in the IP pool. Thank you, @willpower232, for the help.

Now, another issue. Is there any way to scale smtp_server as the receiving seems slow? The mail goes out instantly once it is queued up, but receiving it from the original server is slow. Any fixes?
